src/js/components/Chart.vue

```diff
@@ -0,0 +1,341 @@
+<template>
+  <div class="chart-container">
+    <span class="revenue-label">Total <strong>Revenue</strong></span>
+    <svg class="revenue-line" width="100%" height="310px">
+      <line :y1="Y0" :y2="Y0 + HEIGHT * scale" x1="0" x2="0" />
```

Comment:
I think I moved this outside of the other SVG for layout purposes mainly. Would it be able to appear outside of the other svg if we placed it inside of it?

I think this could work as a border for the label, as long as we can make sure it stays in the same location and the length can stay dynamic based on the total revenue

Reply:
simplified this a bit a moved the `line` inside of the `svg`